docs/conf.py

```diff
@@ -30,7 +47,7 @@
     "base",
     "sphinx-resources",
     "common/README.md",
-    "common/craft-application/how-to/build-remotely.rst",
+    "common/craft-application/how-to-guides/build-remotely.rst",
```

Comment:
```suggestion
    "common/craft-application/how-to/build-remotely.rst",
```
The common docs directory is still named "how-to"